Makefile

```diff
@@ -96,13 +96,13 @@ tidy:
 	go mod tidy
 
 helm:
-	@command -v helm >/dev/null 2>&1 || curl -L https://git.io/get_helm.sh | bash -s -- -v v3.0.0
+	@command -v helm >/dev/null 2>&1 || curl -L https://git.io/get_helm.sh | bash -s -- -v v3.0.1
 
 helm-lint: helm
 	helm lint helm/chart/maesh
 
 k3d:
-	curl -s https://raw.githubusercontent.com/rancher/k3d/v1.3.4/install.sh | TAG=v1.3.4 bash
+	@command -v k3d >/dev/null 2>&1 || curl -s https://raw.githubusercontent.com/rancher/k3d/v1.3.4/install.sh | TAG=v1.3.4 bash
```

Comment:
why is that used?

Reply:
If k3d is already installed, don't re-download and install it again

Reply:
Ah. Thanks!